docs/RELEASE.md

```diff
@@ -35,7 +35,7 @@ docker logout quay.io
 
 Backports are automatically created from PRs against the master branch to the stable branch.
 
-To backport a change, add the `backport` label to the PR that you opened against the master branch.
+To backport a change, add the `backport` (or `backport-squash` but not both) label to the PR that you opened against the master branch.
```

Comment:
@lampajr can you briefly describe the behaviour of each label?

Reply:
Sure, doing it!

Reply:
Added here https://github.com/Hyperfoil/Horreum/pull/1430/files#diff-9c4ae3b5f6675180493e9a1a817baabf4c5314870a3fa67c9c6ab89186046a91R40-R48, let me know if that is clear enough

Reply:
![image](https://github.com/Hyperfoil/Horreum/assets/26715795/108689fb-45f5-4322-980b-00dcfb851b27)

Reply:
LGTM